tests/test_chiral_utils.py

```diff
@@ -0,0 +1,168 @@
+    assert res == set([1])
+
+    mol = Chem.MolFromSmiles(r"FN(Cl)Br")
+    res = chiral_utils.find_chiral_atoms(mol)
+    assert res == set([])
+
+    mol = Chem.MolFromSmiles(r"FS(Cl)Br")
+    res = chiral_utils.find_chiral_atoms(mol)
+    assert res == set([1])
+
+    mol = Chem.MolFromSmiles(r"FP(Cl)Br")
+    res = chiral_utils.find_chiral_atoms(mol)
+    assert res == set([1])
+
+    mol = Chem.AddHs(Chem.MolFromSmiles(r"C1CC2CCC3CCC1N23"))
+    res = chiral_utils.find_chiral_atoms(mol)
+    assert res == set([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
+
+    mol = Chem.AddHs(Chem.MolFromSmiles(r"n1ccccc1"))
+    res = chiral_utils.find_chiral_atoms(mol)
+    assert res == set()
+
+
+def test_find_chiral_bonds():
+
+    mol = Chem.MolFromSmiles(r"FOOF")
+    res = chiral_utils.find_chiral_bonds(mol)
+    assert res == set([])
+
+    mol = Chem.MolFromSmiles(r"FN=NF")
+    res = chiral_utils.find_chiral_bonds(mol)
+    assert res == set([(1, 2)])
+
+    mol = Chem.AddHs(Chem.MolFromSmiles(r"c1cocc1"))
+    res = chiral_utils.find_chiral_bonds(mol)
+    assert res == set([(0, 1), (1, 2), (2, 3), (3, 4), (0, 4)])
+
+    mol = Chem.AddHs(Chem.MolFromSmiles(r"C1=CCC=CO1"))
+    res = chiral_utils.find_chiral_bonds(mol)
+    assert res == set([(0, 1), (3, 4)])
+
+    mol = Chem.AddHs(Chem.MolFromSmiles(r"FNC=O"))
+    res = chiral_utils.find_chiral_bonds(mol)
+    assert res == set([(1, 2)])
+
+    # tautomer of the above
+    mol = Chem.AddHs(Chem.MolFromSmiles(r"O\C=N\F"))
+    res = chiral_utils.find_chiral_bonds(mol)
+    assert res == set([(1, 2)])
+
+    mol = Chem.AddHs(Chem.MolFromSmiles(r"N(C)C(C)=O"))
+    res = chiral_utils.find_chiral_bonds(mol)
+    assert res == set([(0, 2)])
+
+
+def test_find_chiral_atoms_and_bonds_aspirin():
+    mol = Chem.AddHs(Chem.MolFromSmiles(r"CC(=O)OC1=CC=CC=C1C(=O)O"))
+    res = chiral_utils.find_chiral_atoms(mol)
+    assert res == set([0])
+    # benzoic acids should be rotatable - and if even if they were not,
+    # it would be due to intramolecular interactions (so enhancing sampling
+    # over these torsions should be totally fine.)
+    res = chiral_utils.find_chiral_bonds(mol)
+    assert res == set([(4, 5), (5, 6), (6, 7), (7, 8), (8, 9), (4, 9)])
+
+
+def test_find_chiral_atoms_and_bonds_sildenafil():
+    mol = Chem.AddHs(Chem.MolFromSmiles(r"CCCC1=NN(C2=C1N=C(NC2=O)C3=C(C=CC(=C3)S(=O)(=O)N4CCN(CC4)C)OCC)C"))
+    res = chiral_utils.find_chiral_atoms(mol)
+    assert res == set([0, 1, 2, 19, 23, 24, 26, 27, 28, 30, 31, 32])
+    res = chiral_utils.find_chiral_bonds(mol)
+    assert res == set(
+        [
+            (3, 4),
+            (4, 5),
+            (5, 6),
+            (6, 7),
+            (3, 7),
+            (7, 8),
+            (8, 9),
+            (9, 10),
+            (10, 11),
+            (6, 11),
+            (13, 14),
+            (14, 15),
+            (15, 16),
+            (16, 17),
+            (17, 18),
+            (13, 18),
+        ]
+    )
```

Comment:
Specification: This seems surprising to me as desired behavior (catches only aromatic bonds) -- perhaps would be resolved by adding a definition in docstring `find_chiral_bonds`

Reply:
I'll remove this pattern for now - if we need to add it later on it's simple to do.